### ISSUE TITLE
Upload doesn't seem to work on Firefox

### ISSUE THREAD

**Issue by arcaine2** [documize/community#196]
There seems to be no reaction when trying to upload a file on Firefox. Button just highlights but there's no window opened for a file to select. This does work on Chrome, but it would be good to see a progress bar and while uploading a smaller files, ~3MB pdf ,works, uploading a bigger, ~24MB pdf, produces an error:

> goroutine 290 [running]:
github.com/documize/community/edition/logging.Logger.Error(0x0, 0xc00001c640, 0x1, 0xc20d14, 0xe, 0xdb4f60, 0xc0002ae480)
        src/github.com/documize/community/edition/logging/logger.go:56 +0x168
github.com/documize/community/domain/attachment.(*Handler).Add(0xc0004180c0, 0x7f1e80750d98, 0xc00000c768, 0xc0009ffa00)
        src/github.com/documize/community/domain/attachment/endpoint.go:344 +0x7c9
github.com/documize/community/domain/attachment.(*Handler).Add-fm(0x7f1e80750d98, 0xc00000c768, 0xc0009ffa00)
        src/github.com/documize/community/server/routing/routes.go:126 +0x48
net/http.HandlerFunc.ServeHTTP(0xc00026f1c0, 0x7f1e80750d98, 0xc00000c768, 0xc0009ffa00)
        /usr/local/Cellar/go/1.11.5/libexec/src/net/http/server.go:1964 +0x44
github.com/documize/community/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc000114070, 0x7f1e80750d98, 0xc00000c768, 0xc0009ffa00)
        src/github.com/documize/community/vendor/github.com/gorilla/mux/mux.go:159 +0xf1
github.com/documize/community/vendor/github.com/codegangsta/negroni.Wrap.func1(0x7f1e80750d98, 0xc00000c768, 0xc0009ff800, 0xc000821640)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:46 +0x4d
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc00039f020, 0x7f1e80750d98, 0xc00000c768, 0xc0009ff800, 0xc000821640)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:29 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xdb5c80, 0xc00039f020, 0xc00039f080, 0x7f1e80750d98, 0xc00000c768, 0xc0009ff800)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:38 +0x9c
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP-fm(0x7f1e80750d98, 0xc00000c768, 0xc0009ff800)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:38 +0x60
github.com/documize/community/server.(*middleware).Authorize(0xc00026eec0, 0x7f1e80750d98, 0xc00000c768, 0xc000824100, 0xc0007c0e60)
        src/github.com/documize/community/server/middleware.go:228 +0x2d0
github.com/documize/community/server.(*middleware).Authorize-fm(0x7f1e80750d98, 0xc00000c768, 0xc000824100, 0xc0007c0e60)
        src/github.com/documize/community/server/server.go:70 +0x52
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc000476190, 0x7f1e80750d98, 0xc00000c768, 0xc000824100, 0xc0007c0e60)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:29 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xdb5c80, 0xc000476190, 0xc00039f060, 0x7f1e80750d98, 0xc00000c768, 0xc000824100)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:38 +0x9c
github.com/documize/community/vendor/github.com/codegangsta/negroni.(*Negroni).ServeHTTP(0xc0003a3620, 0x7f1e80750d98, 0xc00000c760, 0xc000824100)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:96 +0xee
github.com/documize/community/vendor/github.com/gorilla/mux.(*Router).ServeHTTP(0xc000406700, 0x7f1e80750d98, 0xc00000c760, 0xc000824100)
        src/github.com/documize/community/vendor/github.com/gorilla/mux/mux.go:159 +0xf1
github.com/documize/community/vendor/github.com/codegangsta/negroni.Wrap.func1(0x7f1e80750d98, 0xc00000c760, 0xc0009ff700, 0xc0007c0e40)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:46 +0x4d
github.com/documize/community/vendor/github.com/codegangsta/negroni.HandlerFunc.ServeHTTP(0xc00029e1c0, 0x7f1e80750d98, 0xc00000c760, 0xc0009ff700, 0xc0007c0e40)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:29 +0x4e
github.com/documize/community/vendor/github.com/codegangsta/negroni.middleware.ServeHTTP(0xdb5c80, 0xc00029e1c0, 0xc00029e220, 0x7f1e80750d98, 0xc00000c760, 0xc0009ff700)
        src/github.com/documize/community/vendor/github.com/codegangsta/negroni/negroni.go:38 +0x9c
github.com/documize/community/ve

**Comment by HarveyKandola** [comment]
What's your database?

**Comment by arcaine2** [comment]
```
Database: provider name MySQL
Database: provider version 10.3.11-MariaDB-1:10.3.11+maria~stretch
```

**Comment by HarveyKandola** [comment]
Can you try to increase max_allowed_packet SQL option?

https://docs.documize.com/s/VzO9ZqMOCgABGyfW/installation-guides/d/V2KuM8ICcQABagM5/mysql-specific-database-tuning

**Comment by arcaine2** [comment]
I've set mysql max_allowed_packet to 100M and now uploading files bigger than 16MB works, thanks.

There is still an issue with Firefox.

**Comment by HarveyKandola** [comment]
Fixed in 2.0.3 -- due later today
